Deferred/Deferred.swift

```diff
@@ -116,16 +159,43 @@ public struct Deferred<Value> {
 }
 
 extension Deferred {
+    /**
+    Checks for and returns a determined value.
+    
+    :returns: The determined value, if already filled, or `nil`.
+    */
     public func peek() -> Value? {
         return wait(.Now)
     }
-
+    
+    /**
+    Waits for the value to become determined, then returns it.
+    
+    This is equivalent to unwrapping the value of calling `wait(.Forever)`, but
+    may be more efficient.
+    
+    This getter will unnecessarily block execution. It might be useful for
```

Comment:
There might be some warnings in 2.0. All the `@availability`/`@available` that I know of cause errors.